tendrl/commons/objects/__init__.py

```diff
@@ -75,93 +75,26 @@ def load_definition(self):
 
-        self.updated_at = str(time_utils.now())
-        for item in self.render():
-            '''
-                Note: Log messages in this file have try-except
-                blocks to run
-                in the condition when the node_agent has not been
-                started and
-                name spaces are being created.
-            '''
-            try:
-                Event(
-                    Message(
-                        priority="debug",
-                        publisher=NS.publisher_id,
-                        payload={"message": "Writing %s to %s" %
-                                            (item['key'], item['value'])
-                                 }
-                    )
-                )
-            except KeyError:
-                sys.stdout.write("Writing %s to %s" % (item['key'],
-                                                       item['value']))
-            # convert list, dict (json) to python based on definitions
-            _type = self._defs.get("attrs", {}).get(item['name'],
-                                                    {}).get("type")
-            if _type:
-                if _type.lower() in ['json', 'list']:
-                    if item['value']:
-                        try:
-                            item['value'] = json.dumps(item['value'])
-                        except ValueError as ex:
-                            _msg = "Error save() attr %s for object %s" % \
-                                   (item['name'], self.__name__)
-                            Event(
-                                ExceptionMessage(
-                                    priority="debug",
-                                    publisher=NS.publisher_id,
-                                    payload={"message": _msg,
-                                             "exception": ex
-                                             }
-                                )
-                            )
-            try:
-                NS._int.wclient.write(item['key'], item['value'], quorum=True)
-            except (etcd.EtcdConnectionFailed, etcd.EtcdException):
-                NS._int.wreconnect()
-                NS._int.wclient.write(item['key'], item['value'], quorum=True)
-        if ttl:
-            etcd_utils.refresh(self.value, ttl)
+    def load(self):
+        self.render()
+        key = self.value + '/data'
+        val_str = NS._int.client.read(key).value
+        loc_dict = json.loads(val_str)
+        for attr_name, attr_val in vars(self).iteritems():
+            if not attr_name.startswith('_') and attr_name != "value":
+                _type = self._defs.get("attrs", {}).get(attr_name, {}).get("type")
+                if _type in ['json', 'list', 'Json', 'List', 'JSON', 'LIST']:
+	            setattr(self, attr_name, json.loads(loc_dict[attr_name]))
+                else:
+                    setattr(self, attr_name, loc_dict[attr_name])
```

Comment:
this might not work for DictField https://github.com/Tendrl/commons/blob/master/tendrl/commons/utils/central_store/fields.py#L207

You missed out https://github.com/Tendrl/commons/pull/656/files#diff-fbc8ea2e973a12d3591504a5230f8186L227